lib/seahorn/BvOpSem2.cc

```diff
@@ -185,6 +186,7 @@ struct OpSemVisitorBase {
 
     if (const Constant *cv = dyn_cast<Constant>(&v)) {
       return m_ctx.getConstantValue(*cv);
+      falseE = m_ctx.m_falseE;
```

Comment:
why? won't execute after return

Reply:
this is dead code. I think it is a typo. See above.

Reply:
same as above.